src/templates/LazyTemplateProvider.js

```diff
@@ -46,6 +47,7 @@ export default class LazyTemplateProvider {
   getRadio() { return this._getField(RADIO); }
   getSingleColumn() { return this._getField(SINGLE_COLUMN); }
```

Comment:
Do you want to remove SINGLE_COLUMN and DOUBLE_COLUMN? We probably should from this repo anyways since they were never included in the public release.

Reply:
we decided not to do this